pkg/provisioners/heartbeats/controller/resources/pod.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	"github.com/knative/eventing/pkg/controller"
+
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	image = "gcr.io/plori-nicholss/cmd-f2e9e0e7e0a0f8a66b5b6f8d85f4c98b@sha256:d64b5fd7b4c8c5fdc70cca7732fcf384d3069a4a1762a69e58d701a040345aed"
```

Comment:
An image name like this must not be hardcoded in a source file. This is a no-go in many corporate environments.

Reply:
lolz yeah yeah yeah this is just a demo